src/glkit.cs

```diff
@@ -247,6 +251,7 @@ interface GLKEffectPropertyTexture {
 
 	}
 
+#if !NET
```

Comment:
What's the problem with `GLKEffectPropertyTransform`?

Reply:
it's deprecated, and the keeping it require keeping Matrix[3|4]

Reply:
Pretty much all of `GLKit` is deprecated (in favor or `MetalKit`, which is a very different API). This means that it's not a simple "don't use this class in your code, use this other class instead", it's a "port your entire rendering code to a completely different API". Also GLKit still works, so it's an entirely functional API. For these reasons I believe it's best to keep `GLKEffectPropertyTransform` (and keep/port Matrix3|4).